### PR TITLE
Bugfix/issue 2469 fix alias export

### PR DESCRIPTION
**Pull request by lme-nca**
### Description

Fixes the Vulnerability Alias export in the FPF format export function.

### Addressed Issue

https://github.com/DependencyTrack/dependency-track/issues/2469

### Additional Details

Further discussion on the exact format desired might be useful! This pull request takes a pragmatic solution approach an changes the format to:
`
{
  "vulnerability": {
    "aliases": [
      {
        "ghsaId": "someGhsaId",
        "sonatypeId": "someSonatypeId",
        "snykId": "someSnykId",
        "cveId": "someCveId",
        "vulnDbId": "someVulnDbId",
        "osvId": "someOsvId"
      }
    ]
  }
}
`

I did not provide any test for now as I would like to know what the intended format should look like and possible also in what direction the fix should go.

### Checklist

- [x ] I have read and understand the [contributing guidelines](https://github.com/DependencyTrack/dependency-track/blob/github-templates/CONTRIBUTING.md#pull-requests)
- [ ] This PR fixes a defect, and I have provided tests to verify that the fix is effective
- [ ] This PR implements an enhancement, and I have provided tests to verify that it works as intended
- [ ] This PR introduces changes to the database model, and I have added corresponding [update logic](https://github.com/DependencyTrack/dependency-track/tree/master/src/main/java/org/dependencytrack/upgrade)
- [ ] This PR introduces new or alters existing behavior, and I have updated the [documentation](https://github.com/DependencyTrack/dependency-track/tree/master/docs/_docs) accordingly

### CODE REVIEW

**Comment by stevespringett** [comment]
Note. FPF is versioned, so the FPF version will need to be bumped if changes are made.

**Comment by lme-nca** [comment]
Hi @stevespringett I don't think the FPF version needs to be bumped as the output (with this fix) would then correspond to what was actually intended. Furthermore the version was not bumped when the original change was merged: https://github.com/DependencyTrack/dependency-track/pull/1912/files 

Of course if its still preferred to bump the version now, I could still do this.

**Comment by lme-nca** [comment]
@nscuro @valentijnscholten would appreciate a approval for the workflows to run.

**Comment by valentijnscholten** [comment]
I don't have those powers.

**Comment by nscuro** [comment]
@lme-nca Sorry for the wait, I kicked off the workflows. 

Please do bump the FPF version to 1.2. Additions are backwards-compatible, removals are not. Even if this is a minor bugfix (thanks for spotting it BTW), we should play by the rules of proper versioning.

File: docs/_docs/integrations/file-formats.md [hunk 1]
```diff
@@ -30,6 +30,10 @@ The **VIEW_VULNERABILITY** permission is required to use the findings API.
 > Previous versions of Dependency-Track supported only a single CWE (`cweId` and `cweName` fields respectively) per
 > vulnerability.
 > The `cweId` and `cweName` fields are deprecated and will be removed in a later version. Please use `cwes` instead.
+ 
+> Finding Packaging Format v1.2 was introduced in Dependency-Track v4.7.2.
```
Comment (nscuro) [inline docs/_docs/integrations/file-formats.md:34]: Minor nitpick: It's currently unlikely that we'll release another bugfix version before 4.8.0.

```suggestion
> Finding Packaging Format v1.2 was introduced in Dependency-Track v4.8.0.
```

Comment (lme-nca) [reply docs/_docs/integrations/file-formats.md]: That was just a guess from me :) fixed it: https://github.com/DependencyTrack/dependency-track/pull/2471/commits/dd4680fca6124b4eed9fb5da2d078128e3f156bb

**Comment by lme-nca** [comment]
@nscuro any idea why the tests are failing on this ? `Error:  Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.10.1:compile (default-compile) on project dependency-track: Compilation failure
Error:  /home/runner/work/dependency-track/dependency-track/src/main/java/org/dependencytrack/model/VulnerabilityAlias.java:[205,18] duplicate case label
Error:  `

This compiles and runs fine locally and passed the first time................

**Comment by nscuro** [comment]
No idea why this is happening, it looks fine to me and it builds locally for me as well.

Maybe a problem with how GitHub Actions checked out the repo. Can you try committing a change to the `VulnerabilityAlias` class (e.g. by adding whitespace somewhere)?

**Comment by lme-nca** [comment]
> No idea why this is happening, it looks fine to me and it builds locally for me as well.
> 
> Maybe a problem with how GitHub Actions checked out the repo. Can you try committing a change to the `VulnerabilityAlias` class (e.g. by adding whitespace somewhere)?

Ok it seems, the change i want to introduce is already merged on master now, here: https://github.com/DependencyTrack/dependency-track/commit/33f3fdc7a5020f698abf1a6fc1d6dad0ac03d3f4 ill rebase and push again

**Comment by lme-nca** [comment]
ok rebased master and cleaned-up the commits, it should work now

**Review by nscuro — APPROVED** [review]
Thanks @lme-nca! 🙌

**Pull request merged**